securityconfig/whitelisting_settings.yml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Yes, I can see how this is confusing. I thought that whitelist.yml might be a little vague too, maybe the name should include API somewhere, since this is just for APIs? maybe whitelistApi.yml?

> Since this is mainly use to whitelist APIs maybe whitelisting_settings may be confusing. What about `whitelist.yml`? Let me know what you think.

Yes, I can see how this is confusing. I thought that whitelist.yml might be a little vague too, maybe the name should include API somewhere, since this is just for APIs. Maybe whitelistApi.yml would be better?

Reply:
Have changed it to whitelist.yml. Let me know if you think this is fine, or if whitelist_api.yml would be better

Reply:
Thanks for making the change.